cmd/images.go

```diff
@@ -0,0 +1,225 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"sort"
+	"strings"
+	"time"
+	"unicode"
+
+	"github.com/containers/common/pkg/report"
+	"github.com/containers/podman-bootc/pkg/utils"
+	"github.com/containers/podman/v5/pkg/bindings/images"
+	"github.com/containers/podman/v5/pkg/domain/entities"
+	"github.com/distribution/reference"
+	"github.com/docker/go-units"
+
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+var (
+	imagesCmd = &cobra.Command{
+		Use:   "images",
+		Short: "List bootc images in the local containers store",
+		Long:  "List bootc images in the local container store",
```

Comment:
Why the difference in plurality?
(Do we actually need to duplicate short/long at all with cobra? In short can't we just have `Short:`?)